packages/mdc-button/mdc-button.scss

```diff
@@ -22,6 +22,40 @@
 @import "@material/typography/mixins";
 
 // postcss-bem-linter: define button
+
+.mdc-button {
```

Comment:
These style rules should be put within the `.mdc-button` declaration below, which is the top-level declaration for `mdc-button`. Having an extra one seems redundant, no?